src/stylesheets/components/_subnav.scss

```diff
@@ -1,6 +1,9 @@
 @include govuk-exports("app-subnav") {
 
   .app-subnav {
+    // Since the back to top link is outside the flow of the document
+    // we need to create a space for it.
+    margin-bottom: 100px;
```

Comment:
It might be good to mention in the comment why `100px` was picked as the size of the space

Reply:
just a magic number unfortunately